mmv1/templates/terraform/constants/dbms_private_connection.go.erb

```diff
@@ -0,0 +1,34 @@
+<% unless compiler == "terraformgoogleconversion-codegen" -%>
```

Comment:
It looks like this is copied from the datastream private connection - are we sure this code is all required?